src/components/errorPage/translations.ts

```diff
@@ -0,0 +1,67 @@
+import { StatusCodes } from './statusCodes';
+import { Language } from '../../types/language';
+
+export const translations: Record<
+  Language,
+  Record<
+    StatusCodes,
+    { title: string; description: string; buttonLabel: string }
+  >
+> = {
+  de: {
```

Comment:
It does not need to be in this part but I wonder whether we should add phrase to the components library since we are adding orchestrated components that are much more complex compared to the atom like components we had at CFY.

Reply:
makes sense. I created a ticket: https://autoricardo.atlassian.net/browse/DM-796